### DISCUSSION TITLE
Zooming in on an already rendered map

### DISCUSSION THREAD

**Discussion by mishuha** [time4tea/gopro-dashboard-overlay#155] [Ideas]
Hello.

I know there are zoom levels implemented through the maptiler API. But the font size of the displayed labels on the map does not change with the scale. It's too small for my eyes and YouTube's encoders also make the labels blurry. And frankly, I don't want to create a map tile server just to change the font size. I'd prefer to decrease zoom level and resize image to fit the map into the same borders.

Is it possible to change the size of an already drawn map without losing the coordinates? Perhaps this would be a nice feature.

**Comment by time4tea** [comment]
Hi - Thanks for this suggestion - I'm looking into possible new ways to render the maps.
